choice.rb

```diff
@@ -2,12 +2,23 @@
 require './teacher'
 require './book'
 require './rental'
+require 'json'
+require './data'
 
-class MenuChoice
+class HandleMenuChoice
   def initialize
     @people = HandlePerson.new
+    @people.read_people_json
     @books = HandleBooks.new
+    @books.read_books_json
     @rentals = HandleRentals.new
+    @rentals.read_rentals_json(@people.people, @books.books)
+  end
+
+  def saving_exit
+    puts 'Saving'
+    @persistor = Data.new
+    @persistor.save(people: @people.people, books: @books.books, rentals: @rentals.rentals)
   end
```

Comment:
- [ ] Currently, the data is not saving and it's giving an error while doing that. I'm attaching the screenshot of that as well, please take a look and fix that.

![Screenshot from 2022-01-14 19-46-17](https://user-images.githubusercontent.com/22434660/149530020-28224cb5-d19e-49af-a9be-daddf58cafe1.png)